css/style.css

```diff
@@ -542,12 +593,31 @@ button:hover {
   align-self: center;
 }
 
+@media (max-width: 56.25em) {
+  .cases__content > h2 {
+    font-size: 2rem;
+    margin: 0;
+  }
+}
+
+@media (max-width: 37.5em) {
+  .cases__content > h2 {
+    font-size: 1.5rem;
+  }
+}
+
 .cases__devices {
   list-style: none;
   display: flex;
   justify-content: space-between;
 }
 
+@media (max-width: 37.5em) {
+  .cases__devices {
+    flex-direction: column;
+  }
+}
```

Comment:
Good, this looks great! 

Watch out for the spacing around that section - there is no padding at the top and around the headline, and a lot of whitespace at the bottom. Try and even the space out.

Reply:
Yes, I will add padding.